textplot/text.py

```diff
@@ -28,38 +28,54 @@ def from_file(cls, path):
             path (str): The file path.
         """
 
-        with open(path, 'r', errors='replace') as f:
-            return cls(f.read())
+        # with open(path, 'r', errors='replace') as f:
+        #     return cls(f.read())
 
```

Comment:
I assume this change is intended to support list of texts as input rather than a text file, but we should probably aim to maintain support for the original input as well as other potential input formats, e.g. directory containing multiple txt files, list of documents as strings, etc.